.circleci/config.yml

```diff
@@ -441,3 +444,35 @@ workflows:
               ignore: /.*/
             tags:
               only: /^[0-9]+\.[0-9]+\.[0-9]+$/
+
+  'Build Earth Images':
+    jobs:
+      - docker/publish:
+          docker-password: DOCKER_HUB_PW
+          docker-username: DOCKER_HUB_USER
+          image: 'prefecthq/apollo'
+          path: 'server/services/apollo'
+          tag: $CIRCLE_SHA1,latest,master
+          filters:
+            branches:
+              only: master
```

Comment:
Can we change this to filter based on tag, similar to the `release_to_pypi` job?  I don't think we need to rebuild these image with every PR, only releases

Reply:
@cicdw can we make it master so we can see it run then we can update it to only on releases?

Reply:
Yup for sure, that makes sense